control_toolbox/include/control_toolbox/low_pass_filter.hpp

```diff
@@ -124,6 +124,8 @@ class LowPassFilter
   double b1_; /** feedforward coefficient. */
   /** internal data storage (double). */
   double filtered_value, filtered_old_value, old_value;
+  /** internal data storage (std::vector<double>). */
+  std::vector<double> vector_filtered_value, vector_filtered_old_value, vector_old_value;
```

Comment:
can't we use the template type here, instead of adding additional member variables?

Reply:
I agree. Much better

Reply:
We should alsp start thinking of adding underscore to the member variables